authentication/auth.go

```diff
@@ -235,6 +236,15 @@ func launcherSchemaFromURL(url string) (launcherSchema surveys.LauncherSchema) {
 		panic(err)
 	}
 
+	if resp.StatusCode != 200 {
```

Comment:
Since your returning here if response is != 200, wonder if it's worth moving it above where the respnseBody is read? Since there's no point reading the body if we're not doing anything with it?